### PR TITLE
issue10: Adding simple interface for lines

### PR DESCRIPTION
**Pull request by pthien**
Change default data plotted to consist of 3 data sets - 3 random walks, and a 3D sinc function

### CODE REVIEW

File: src/rendering/mod.rs [hunk 1]
```diff
@@ -10,6 +10,13 @@ pub struct Vertex {
     pub color: [f32; 4],
     pub size: f32,
 }
+
+pub struct Line {
+    pub indicies: Vec<u32>,
```
Comment (a7g4) [inline src/rendering/mod.rs:15]: You mentioned potentially moving this out of the struct. Can you make an issue for that?

File: src/main.rs [hunk 2]
```diff
@@ -142,13 +161,21 @@ fn main() {
...
                     .device
                     .create_command_encoder(&wgpu::CommandEncoderDescriptor { label: None });
-                    renderer.render(&mut commands, &frame.view, &rendering::defaults::axes(), &rendering::defaults::render_all_vertices(&rendering::defaults::axes()), true);
-                renderer.render(&mut commands, &frame.view, &vertices, &indecies, false);
+                renderer.render(&mut commands, &frame.view, &rendering::defaults::axes(), &rendering::defaults::render_all_vertices(&rendering::defaults::axes()), true);
+                //renderer.render(&mut commands, &frame.view, &vertices, &indecies, false);
+                for i in 0..lines.len() {
+                    let v = &lines[i].verticies;
+                    let i = &lines[i].indicies;
+                    renderer.render(&mut commands, &frame.view, v, &i, false);
```
Comment (a7g4) [inline src/main.rs:169]: This is going to render individual points on the line as unconnected dots; is that your intent?

(And if it is, do you want to eventually support the connections? If so, make an issue 😃 )

**Review by a7g4 — APPROVED** [review]

File: src/main.rs (hunk 2 above)
Comment (pthien) [reply src/main.rs:169]: done. https://github.com/dr-sg/rscat/issues/12

File: src/rendering/mod.rs (hunk 1 above)
Comment (pthien) [reply src/rendering/mod.rs:15]: done https://github.com/dr-sg/rscat/issues/13

**Pull request merged**